File: backend/Modulos/models/movimientosFarmaciaDetDevol.go
package models

// MovimientosFarmaciaDetDevol is...
type MovimientosFarmaciaDetDevol struct {
	MovimFarDetID      int    `json:"movimfardetid"`
	MovimFarDetDevolID int    `json:"movimfardetdevolid"`
	MovTipo            int    `json:"tipomov"`
	FechaMovDevol      string `json:"fechamovdevol"`
	CantidadDevol      int    `json:"cantidaddevol"`
	ResponsableNom     string `json:"responsablenom"`
	CuentaCargoID      int    `json:"cuentacargoid"`
	CantidadDevolTot   int    `json:"cantidaddevoltot"`
}

File: frontend/src/app/models/entity/MovimientosFarmaciaDetDevol.ts
export class MovimientosFarmaciaDetDevol {
    constructor (
        public movimfarid         ?: number,
        public movimfardetid      ?: number, 
        public movimfardetdevolid ?: number,
        public tipomov            ?: number,
        public fechamovdevol      ?: string,
        public cantidaddevol      ?: number,
        public responsablenom     ?: string,
        public cuentacargoid      ?: number,
        public cantidaddevoltot   ?: number,
        public hdgcodigo          ?: number,
        public esacodigo          ?: number,
        public cmecodigo          ?: number,
        public servidor           ?: string,
     ) {}
}